enterprise/app/trends/trends.tsx

```diff
@@ -312,23 +320,28 @@ export default class TrendsComponent extends React.Component<Props, State> {
                 data={this.state.timeKeys}
                 dataSeries={[
                   {
-                    name: "failed builds",
+                    name: `failed builds (${format.count(getTotal(this.getStats(), (stat) => +(stat.failedBuilds ?? 0)))})`,
                     extractValue: (tsMillis) => +(this.getStat(tsMillis).failedBuilds ?? 0),
                     formatHoverValue: (value) => (value || 0) + " failed builds",
                     onClick: this.onBarClicked.bind(this, "", ""),
                     stackId: "builds",
                     color: ChartColor.RED,
                   },
                   {
-                    name: "successful builds",
+                    name: `successful builds (${format.count(getTotal(this.getStats(), (stat) => +(stat.successfulBuilds ?? 0)))})`,
                     extractValue: (tsMillis) => +(this.getStat(tsMillis).successfulBuilds ?? 0),
                     formatHoverValue: (value) => (value || 0) + " successful builds",
                     onClick: this.onBarClicked.bind(this, "", ""),
                     stackId: "builds",
                     color: ChartColor.GREEN,
                   },
                   {
-                    name: "average build duration",
+                    name: `average build duration (${format.durationUsec(
+                      getAverage(
+                        this.getStats().filter((stat) => stat.completedInvocationCount),
+                        (stat) => +(stat.totalBuildTimeUsec ?? 0) / +(stat.completedInvocationCount ?? 0)
```

Comment:
i guess it only matters when there are no builds, but i think you want a 1 in this denominator.

Reply:
I think this case should be fine since we're filtering out the stats where the denominator is zero. It's possible the numerator will be > 0 and the denominator is 0 but that seems like it'd be a bug in the query and I'd be fine just ignoring those cases - wdyt?

The cases below aren't handling NaN properly though - updated. (I didn't see NaNs in my local server but I think it's only because the formatting functions treat falsy values specially, and NaN is falsy, but that's not really obvious at all haha)